router/psychologists.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const controller = require('../controllers/psychologists')
+
+const router = express.Router()
+
+const {
+  getPsychologists,
+  createPsychologist,
+  deletePsychologist,
+  updatePsychologist
+} = controller
+
+router.post('/', createPsychologist)
+router.put('/', updatePsychologist)
+router.delete('/', deletePsychologist)
```

Comment:
you have to pass the id through the path in `deletePsychologist` and `updatePsychologist` endpoints